scripts/train_linker.py

```diff
@@ -0,0 +1,72 @@
+from sklearn.ensemble import GradientBoostingClassifier
+from sklearn.metrics import accuracy_score, classification_report
+from joblib import dump, load
+import jsonlines
+import datetime
+from scispacy import umls_semantic_type_tree
+from linking import featurizer
+import argparse
+from tqdm import tqdm
+
+
+def main(data_path: str):
+    train = []
+    dev = []
+    test = []
+    root = ''
+    files = [f'{data_path}/train.jsonl', f'{data_path}/dev.jsonl', f'{data_path}/test.jsonl']
+    lists = [train, dev, test]
+    # limits = [40000000, 4000000, 4000000]
+    limits = [4000000, 0, 4000000]
```

Comment:
Why are we not reading the dev data, but the test data only?

Reply:
Just to save time, the dataset is very large compared to the features we have. For validation, sklearn classifiers automatically split the training set into train and dev